### DISCUSSION TITLE
how can i change the kitty scroll up and down to vim key bindings one like in vim we scroll down with j and scroll up with k so same thing i want

### DISCUSSION THREAD

**Discussion by Greatz08** [kovidgoyal/kitty#7181] [Q&A]
please guide in detail like what i have to write in kitty conf to change scroll up and down functionality of kitty
Thankyou :-))

**Comment by cybardev** [comment]
The guide is here in the docs: https://sw.kovidgoyal.net/kitty/conf/#scrolling

The issue with directly binding `j` and `k` though, is that you won't be able to type those letters if needed I'm assuming, unless you bind them to something else, which doesn't make sense in my opinion.

That's why the docs map it to `Ctrl + Shift + j` and `Ctrl + Shift + k` instead.
